test/stream-collateral/valid-default/expected-content/test/asyncNegative_strict_mode.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+ function Test262Error(message) {
```

Comment:
There's a space in front of `function`, maybe it's intentional? Either way, it's not a blocker, can be fixed or not.

Reply:
It's not intentional as far as I can tell, but it is an artifact of the previously-existing implementation. Happy to address it here, but it should be done with a dedicated commit